evap/static/js/datagrid.js

```diff
@@ -0,0 +1,314 @@
+class DataGrid {
+    // Use init instead of constructor to enable access to this before calling super
+    init({storageKey, container, searchInput}) {
+        this.storageKey = storageKey;
+        this.container = container;
+        this.sortableColumns = Array.from(this.sortableHeaders.keys());
```

Comment:
Is there a benefit to keeping sortableColumns and sortableHeaders separate? I think I'd prefer to give each column a header, and not having two seemingly-but-in-fact-not independent datastructures.